src/OrchardCore.Modules/OrchardCore.ContentTypes/Editors/ContentTypeSettingsDisplayDriver.cs

```diff
@@ -39,25 +49,58 @@ public override async Task<IDisplayResult> UpdateAsync(ContentTypeDefinition con
 
             if (await context.Updater.TryUpdateModelAsync(model, Prefix))
             {
-                context.Builder.Creatable(model.Creatable);
-                context.Builder.Listable(model.Listable);
-                context.Builder.Draftable(model.Draftable);
-                context.Builder.Versionable(model.Versionable);
-                context.Builder.Securable(model.Securable);
-                context.Builder.WithDescription(model.Description);
-
                 var stereotype = model.Stereotype?.Trim();
+                context.Builder.WithDescription(model.Description);
                 context.Builder.Stereotype(stereotype);
 
                 if (!IsAlphaNumericOrEmpty(stereotype))
                 {
                     context.Updater.ModelState.AddModelError(nameof(ContentTypeSettingsViewModel.Stereotype), S["The stereotype should be alphanumeric."]);
                 }
+
+                var options = GetOptions(contentTypeDefinition, stereotype);
+
+                Apply(context, model, options);
             }
 
             return Edit(contentTypeDefinition);
         }
 
+        private static void Apply(UpdateTypeEditorContext context, ContentTypeSettingsViewModel model, ContentTypeDefinitionDriverOptions options)
+        {
+            if (options.ShowVersionable)
+            {
+                context.Builder.Versionable(model.Versionable);
+            }
+            if (options.ShowCreatable)
+            {
+                context.Builder.Creatable(model.Creatable);
+            }
+            if (options.ShowSecurable)
+            {
+                context.Builder.Securable(model.Securable);
+            }
+            if (options.ShowListable)
+            {
+                context.Builder.Listable(model.Listable);
+            }
+        }
+
+        private ContentTypeDefinitionDriverOptions GetOptions(ContentTypeDefinition contentTypeDefinition, string sterotype)
+        {
+            if (sterotype != null && _options.Stereotypes.TryGetValue(sterotype, out var stereotypesOptions))
+            {
+                return stereotypesOptions;
+            }
+
+            if (contentTypeDefinition.Name != null && _options.Stereotypes.TryGetValue(contentTypeDefinition.Name, out var typeOptions))
+            {
+                return typeOptions;
+            }
```

Comment:
I'd switch these up, to make content type options of higher priority. Otherwise, I can set options for the `Content` stereotype, overriding everything else.

Reply:
`Content` is not a stereotype in the settings. so it would not cause an error. But, we can switch them

Reply:
swapped.